dev/com.ibm.ws.javaee.version/src/com/ibm/ws/javaee/version/PagesVersion.java

```diff
@@ -8,13 +10,10 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.jsp.webcontainerext;
 
 /**
- *
- */
-public interface JspVersionFactory {
-    
-    public JspVersion getJspVersion();
-
+*  Holder class for version info.
+*/
+public class PagesVersion {
```

Comment:
PagesVerion or JspVersion?

Reply:
I think PagesVersion is fine it could easily be related back to `JSP` (Java Server Pages) or to Jakarta  Server Pages in my opinion